_posts/wfh1972/2016-02-15-wfh1972-RefactoredTetris.md

```diff
@@ -0,0 +1,17 @@
+---
+author: wfh1972
+layout: post
+title: "Wayne Homan's broken refactored tetris"
+---
+
+<iframe src="https://trinket.io/embed/python/35890baf32" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
```

Comment:
Wayne, I don't see large amounts of commented code on this.  Do you maybe need to press 'Save' in trinket to save your draft?